boards/arm/msp_exp432p401r_launchxl/doc/msp_exp432p401r_launchxl.rst

```diff
@@ -0,0 +1,112 @@
+.. _msp_exp432p401r_launchxl:
+
+MSP-EXP432P401R LaunchXL
+########################
+
+Overview
+********
+
+The SimpleLink MSP‐EXP432P401R LaunchPad development kit is an easy-to-use evaluation
+module for the SimpleLink MSP432P401R microcontroller. It contains everything needed to start
+developing on the SimpleLink MSP432 low-power + performance ARM |reg| 32-bit Cortex |reg|-M4F
+microcontroller (MCU). The MSP432P401R device supports low-power applications requiring increased CPU
```

Comment:
"increased" implies comparison with something else, but what are you comparing to?  And I'm not sure what "32-bit performance" means.